### PR TITLE
Native schema updates

### PR DESCRIPTION
**Pull request by giorgosp**
### Background

Syncing native schema changes

### Testing

* <Testing steps>

### CODE REVIEW

File: schemas/logs/aws/alb.yml [hunk 1]
```diff
@@ -23,13 +24,13 @@ fields:
     type: string
   - name: clientPort
     description: The port of the requesting client.
-    type: int
+    type: bigint
```
Comment (kostaspap) [inline schemas/logs/aws/alb.yml:27]: It seems that these types are changing again. I know that in theory it shouldn't matter, but is that expected to happen again @alxarch

**Review by kostaspap — COMMENTED** [review]
Tests seem to be failing. @alxarch  do you have more context on that? This PR shouldn't affect them.

File: schemas/logs/aws/alb.yml (hunk 1 above)
Comment (alxarch) [reply schemas/logs/aws/alb.yml:27]: Yes this is the `push` part of the mage PR. This is expected.

**Comment by giorgosp** [comment]
I'm closing this PR until the issue with tests is fixed. @alxarch has a task for it.
Syncing schemas is not urgent, it can be done after the issue is fixed.

**Pull request closed without merging**